RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -376,44 +376,82 @@
+    mvaName_wGwGSF_BL = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_wGwGSF_BL",
+    mvaName_NoEleMatch_woGwoGSF_EC = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_NoEleMatch_woGwoGSF_EC",
+    mvaName_NoEleMatch_wGwoGSF_EC = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_NoEleMatch_wGwoGSF_EC",
+    mvaName_woGwGSF_EC = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_woGwGSF_EC",
+    mvaName_wGwGSF_EC = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_wGwGSF_EC"
+)
+_vLooseElectronRejectionMappingMVA6v3 = copy.deepcopy(hpsPFTauDiscriminationByMVA6VLooseElectronRejection.mapping)
+_vLooseElectronRejectionMappingMVA6v3[0].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_NoEleMatch_woGwoGSF_BL_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[1].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_NoEleMatch_wGwoGSF_BL_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[2].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_woGwGSF_BL_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[3].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_wGwGSF_BL_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[4].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_NoEleMatch_woGwoGSF_EC_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[5].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_NoEleMatch_wGwoGSF_EC_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[6].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_woGwGSF_EC_WPeff98"
+_vLooseElectronRejectionMappingMVA6v3[7].cut = "RecoTauTag_antiElectronMVA6v3_noeveto_gbr_wGwGSF_EC_WPeff98"
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6VLooseElectronRejection,
+    mapping = _vLooseElectronRejectionMappingMVA6v3
+)
+_looseElectronRejectionMappingMVA6v3 = copy.deepcopy(_vLooseElectronRejectionMappingMVA6v3)
+for m in _looseElectronRejectionMappingMVA6v3:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff90")
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6LooseElectronRejection,
+    mapping = _looseElectronRejectionMappingMVA6v3
+)
+_mediumElectronRejectionMappingMVA6v3 = copy.deepcopy(_vLooseElectronRejectionMappingMVA6v3)
+for m in _mediumElectronRejectionMappingMVA6v3:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff80")
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6MediumElectronRejection,
+    mapping = _mediumElectronRejectionMappingMVA6v3
+)
+_tightElectronRejectionMappingMVA6v3 = copy.deepcopy(_vLooseElectronRejectionMappingMVA6v3)
+for m in _tightElectronRejectionMappingMVA6v3:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff70")
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6TightElectronRejection,
+    mapping = _tightElectronRejectionMappingMVA6v3
+)
+_vTightElectronRejectionMappingMVA6v3 = copy.deepcopy(_vLooseElectronRejectionMappingMVA6v3)
+for m in _vTightElectronRejectionMappingMVA6v3:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff60")
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6VTightElectronRejection,
+    mapping = _vTightElectronRejectionMappingMVA6v3
+)
```

Comment:
this should be done right after hpsPFTauDiscriminationByMVA6VLooseElectronRejection is defined and before copies of it are made
```suggestion
```
see the replacement (partial) above